launchable/commands/record/tests.py

```diff
@@ -362,6 +416,13 @@ def send(payload: Dict[str, Union[str, List]]) -> None:
 
                 res.raise_for_status()
 
+                # If don’t override build, test session and session_id, build and test session will be made per chunk request.
+                if is_no_build:
+                    self.build_name = res.json().get("build", {}).get("build", NO_BUILD_BUILD_NAME)
+                    self.test_session_id = res.json().get("testSession", {}).get("id", NO_BUILD_TEST_SESSION_ID)
+                    self.session_id = "builds/{}/test_sessions/{}".format(self.build_name, self.test_session_id)
```

Comment:
[must]
I think using `session_id` as a URL path is not appropriate variable name.
Probably something like `subpath` or `path` is better.

Reply:
ref: https://github.com/launchableinc/cli/pull/537#discussion_r1099746647

Reply:
I mean, using variable name `session_id` as subpath is misleading.